app/src/main/java/com/example/solidarity/fragments/EventsFragment.java

```diff
@@ -121,12 +167,53 @@ public void done(List<Event> events, ParseException e) {
                     return;
                 }
                 adapter.clear();
-                adapter.addAll(events);
+                for (Event ev: events) {
+                    getDist(latitude, longitude, ev.getLocation(), distances, ev);
+                }
                 swipeContainer.setRefreshing(false);
-
-
             }
         });
+    }
 
+    public void getDist(String fromLat, String fromLong, String toAddress,
+                        final ArrayList<Integer> distances, final Event event) {
+        String URL_4 = "https://maps.googleapis.com/maps/api/distancematrix/json?uni" +
```

Comment:
Why is the variable named URL_4 ? what is 4 and why is it all caps?

Reply:
I was testing out different URL formats, and forgot to rename this- sorry about that!